src/interfaces/node.cpp

```diff
@@ -183,6 +183,17 @@ class NodeImpl : public Node
         LOCK(::cs_main);
         return ::ChainActive().Height();
     }
+
+    uint256 getBestBlockHash() override
+    {
+        const CBlockIndex* tip = nullptr;
+        {
+            LOCK(::cs_main);
+            tip = ::ChainActive().Tip();
+        }
+        return (tip) ? tip->GetBlockHash() : Params().GenesisBlock().GetHash();
```

Comment:
In commit "Added best block hash to the NotifyHeaderTip and NotifyBlockTip signals." (4948d4ee50d9960b23f43e14938845b0bb21772b)

Maybe drop extra parentheses around (tip)

Reply:
done